detox/src/logger/utils/DetoxLogFinalizer.test.js

```diff
@@ -172,21 +172,10 @@ describe('DetoxLogFinalizer', () => {
       expect(fs.existsSync(temporaryFiles[2])).toBe(false);
     });
 
-    it('should not create logs for successful run when configured', async () => {
+    it('should create logs even for successful run when configured to keep failing logs', async () => {
```

Comment:
Please mind that this scenario is being checked for "finalizeSync" (which is an emergency substitute for the "finalize").